### PR TITLE
Graphql mutation check

### PR DESCRIPTION
**Pull request by paulbrighton**

### CODE REVIEW

File: pages/article.js [hunk 1]
```diff
@@ -19,16 +33,31 @@ class Articles extends Component {
...
     `
   }
 
+  postComment (event, newComment) {
+    event.preventDefault()
+    const form = event.target
+
+    newComment({
+      variables: {
+        articleId: this.props.url.query.id,
```
Comment (krisquigley) [inline pages/article.js]: This should be slug, and you need to add the I'm age variable too

File: pages/article.js [hunk 2]
```diff
@@ -39,6 +68,24 @@ class Articles extends Component {
         } else {
           return <Layout {...this.props}>
             <Article article={article} />
+            <Mutation mutation={this.newComment}
+              update={(cache, { data: { newComment } }) => {
```
Comment (krisquigley) [inline pages/article.js]: I would get the mutation working without this update function first. Can add it in after. The mutation should add the comment. But you'll need to refresh the page to see it at first

File: scss/components/_c-article.scss [hunk 3]
```diff
@@ -1,6 +1,10 @@
 .c-article {
   width: 100%;
 
+  &__header {
+    clear: both;
```
Comment (krisquigley) [inline scss/components/_c-article.scss]: Shouldn't need clears in modern CSS development

File: components/Article.js [hunk 4]
```diff
@@ -23,8 +22,7 @@ class Article extends Component {
           <MarkdownPreview value={article.text} />
         </div>
       </div>
-      <UserComments comments={article.comments} />
-      <a className='c-user-comments__back-btn'>Back to Article List</a>
+      <Link href='/index'><a className='c-user-comments__back-btn' onClick={click}>Back to Article List</a></Link>
```
Comment (krisquigley) [inline components/Article.js]: What does `onClick={click}` do here?

Comment (paulbrighton) [reply components/Article.js]: I had used it in my nav, <li><Link href='/index'><a onClick={click}>Home</a></Link></li> so thought that was correct

Comment (paulbrighton) [reply components/Article.js]: Not needed, removed.

File: components/Article.js [hunk 5]
```diff
@@ -23,8 +21,9 @@ class Article extends Component {
           <MarkdownPreview value={article.text} />
         </div>
       </div>
-      <UserComments comments={article.comments} />
-      <a className='c-user-comments__back-btn'>Back to Article List</a>
+      <div class='c-user-comments__articles-link'>
+        <Link href='/index'><a className='c-user-comments__back-btn'>Back to Article List</a></Link>
```
Comment (krisquigley) [inline components/Article.js]: I would probably just use `/` for the href, otherwise the url will be `/index`

**Pull request merged**